content/articles/getting-started-with-kali-linux/index.md

```diff
@@ -0,0 +1,290 @@
+### Getting started with Kali Linux
+
+**Kali Linux** is the most popular penetration testing platform that provides a variety of security auditing tools used by security specialists and hackers in day-to-day encounters.
+
+Tools available on Kali Linux enable the user to gather information, perform exploits and prevent their devices from exploits.
+
+### Installing Kali Linux
+
+For beginners, it is advised to have your Kali Linux in a **virtual box**. 
```

Comment:
Why is it advisable to use a virtual box for beginners? You need to explain why.